config/samples/hybris_v1alpha1_hybrisapp.yaml

```diff
@@ -3,8 +3,11 @@ kind: HybrisApp
 metadata:
   name: hybrisapp-sample
 spec:
-  baseImageName: hybris-base
+  baseImageName: hybris-base-for-testing
   baseImageTag: v0.1
   sourceRepoRef: master
-  sourceRepoURL: https://github.com/redhat-sap/sap-commerce-app-example.git
+  sourceRepoURL: https://github.com/faizalak/sap-commerce-app-example.git
```

Comment:
Is this change intended?
If it's just for a testing, perhaps a new "testing" sample file could be created?

Reply:
sap-commerce-app-example is a sample user-created CR for testing the operator. The repo now has a new property file, which is one of the changes to test. This will not impact runtime.

Reply:
I see, thanks for the explanation.